readthedocs/templates/projects/domain_list.html

```diff
@@ -12,11 +12,22 @@
 {% block project_edit_content_header %}{% trans "Domains" %}{% endblock %}
 
 {% block project_edit_content %}
-  <p class="help_text">
-    {% blocktrans trimmed with docs_url='https://docs.readthedocs.io/page/custom_domains.html' %}
-        Configuring a custom domain allows you to serve your documentation from a domain other than "{{ default_domain }}". <a href="{{ docs_url }}">Learn more</a>.
-    {% endblocktrans %}
-  </p>
+  {% if superproject %}
+    {% url "projects_detail" superproject.slug as superproject_url %}
+    <p class="help_text">
+      {% blocktrans trimmed with superproject=superproject.name superproject_url=superproject_url domain=project.subdomain %}
+        This project is a subproject of <a href="{{ superproject_url }}">{{ superproject }}</a>,
+        its documentation will always be served from the <code>{{ domain }}</code> domain.
+        <a href="https://docs.readthedocs.io/page/subprojects.html#custom-domain-on-subprojects">Learn more</a>.
+      {% endblocktrans %}
+    </p>
+  {% else %}
+    <p class="help_text">
+      {% blocktrans trimmed with docs_url='https://docs.readthedocs.io/page/custom_domains.html' %}
+          Configuring a custom domain allows you to serve your documentation from a domain other than "{{ default_domain }}". <a href="{{ docs_url }}">Learn more</a>.
+      {% endblocktrans %}
```

Comment:
so, not sure if we should show both messages when this is a subproject.

Reply:
I think it makes sense to just show one block. We have discussed on another issue that a better UX would be to show the form as disabled, but we can come back to this as well. There are maybe a few patterns to figure out there.